examples/language/lib/patterns/pattern_types.dart

```diff
@@ -0,0 +1,262 @@
+// ignore_for_file: duplicate_variable_pattern, duplicate_definition, unused_local_variable, dead_code, constant_pattern_never_matches_value_type, unnecessary_null_check_pattern, pattern_never_matches_value_type, unreachable_switch_case, unused_element
```

Comment:
Let's split these up to make them easier to read and edit. Preferably we can remove more of them over time. This already removes `unused_element`.

```suggestion
// ignore_for_file: duplicate_variable_pattern, duplicate_definition, unused_local_variable
// ignore_for_file: dead_code, constant_pattern_never_matches_value_type, unnecessary_null_check_pattern
// ignore_for_file: pattern_never_matches_value_type, unreachable_switch_case
```